<launch>

    <node pkg="festo_ovem" type="festo_ovem_node" name="node" launch-prefix="ethercat_grant" ns="vacuum"> 
        <param name="interface_name" value="enp3s0" />
    </node>

</launch>
